packages/config/src/validation.ts

```diff
@@ -120,20 +120,88 @@ export const isValidBrowserList = (_key: string, browsers: any): ErrResult | tru
   const optionalKeys = ['runMode', 'openMode']
+  const experimentalOptions = ['experimentalStrategy', 'experimentalOptions']
+  const experimentalStrategyOptions = ['detect-flake-but-always-fail', 'detect-flake-and-pass-on-threshold']
+
   const isValidRetryValue = (val: any) => _.isNull(val) || (Number.isInteger(val) && val >= 0)
   const optionalKeysAreValid = (val: any, k: string) => optionalKeys.includes(k) && isValidRetryValue(val)
 
   if (isValidRetryValue(value)) {
     return true
   }
 
-  if (_.isObject(value) && _.every(value, optionalKeysAreValid)) {
-    return true
+  if (_.isObject(value)) {
+    const traditionalConfigOptions = _.omit(value, experimentalOptions)
+    const experimentalConfigOptions = _.pick<any>(value, experimentalOptions)
+
+    const isTraditionalConfigValid = _.every(traditionalConfigOptions, optionalKeysAreValid)
+
+    // if optionalKeys are only present and are valid, return true.
+    // The defaults for 'experimentalStrategy' and 'experimentalOptions' are undefined, but the keys exist, so we need to check for this
+    if (isTraditionalConfigValid && !Object.keys(experimentalConfigOptions).filter((key) => experimentalConfigOptions[key] !== undefined).length) {
+      return true
+    }
+
+    // check experimental configuration. experimentalStrategy MUST be present if experimental config is provided and set to one of the provided enumerations
+    if (experimentalConfigOptions.experimentalStrategy) {
+      // make sure the strategy provided is one of our valid enums
+      const isValidStrategy = experimentalStrategyOptions.includes(experimentalConfigOptions.experimentalStrategy)
+
+      // if a strategy is provided, and traditional options are also provided, such as runMode and openMode, then these values need to be booleans
+      const openAndRunModeConfigOptions = _.pick(value, optionalKeys)
+      const isValidRunAndOpenModeConfigWithStrategy = _.every(openAndRunModeConfigOptions, _.isBoolean)
+
+      // if options aren't present (either undefined or null) or are configured correctly, return true
+      if (isValidStrategy && isValidRunAndOpenModeConfigWithStrategy && (
+        experimentalConfigOptions.experimentalOptions == null ||
+        isValidExperimentalRetryOptionsConfig(experimentalConfigOptions.experimentalOptions, experimentalConfigOptions.experimentalStrategy))) {
+        return true
+      }
+    }
   }
 
-  return errMsg(key, value, 'a positive number or null or an object with keys "openMode" and "runMode" with values of numbers or nulls')
+  return errMsg(key, value, 'a positive number or null or an object with keys "openMode" and "runMode" with values of numbers, booleans, or nulls, or experimental configuration with key "experimentalStrategy" with value "detect-flake-but-always-fail" or "detect-flake-and-pass-on-threshold" and key "experimentalOptions" to provide a valid configuration for your selected strategy')
```

Comment:
Hm, yea I don't know how I would simplify this in this current format. I guess our options would be either:
1. Link to a doc instead of explaining this all in-line
2. Break up the error message into conditional parts, so that it isn't just one error message to cover all cases.

I think I lean towards the latter.